platform/cisco-8000/rules.mk

```diff
@@ -0,0 +1,14 @@
+##############################################################
+## Top level platform rules.mk
+##############################################################
+CISCO_PLATFORM_BRANCH := cisco-8000/master
+CISCO_PLATFORM_URL := https://$(GITUSER):$(GITTOKEN)@Cisco-sonic-8000/platform-cisco-8000.git
```

Comment:
Does it miss the "github.com/"? I guess the format is as below, correct me if wrong.
https://$(GITUSER):$(GITTOKEN)@github.com/Cisco-sonic-8000/platform-cisco-8000.git

Reply:
Cisco-8000-sonic or Cisco-sonic-8000?